### DISCUSSION TITLE
Can the D435i replace the PS3?

### DISCUSSION THREAD

**Discussion by Carloslius** [jyjblrd/Mocap-Drones#30] [General]
Can I use the infrared camera in Realsense's D435i instead of the PS3?

**Comment by jyjblrd** [comment]
If it can be used as a usb webcam then yes, but those cameras are pretty expensive and this doesn't utilize their depth sensing at all. Maybe if the cameras have stereo vision and you can access both video streams you can utilize both cameras.
